Modules/Forum/classes/GUI/class.ilForumThreadFormGUI.php

```diff
@@ -155,7 +155,7 @@ protected function initForm()
             $this->addItem($captcha);
         }
 
-        if (\ilForumPostDraft::isSavePostDraftAllowed()) {
+        if (\ilForumPostDraft::isSavePostDraftAllowed() && $this->user->getId() != ANONYMOUS_USER_ID) {
```

Comment:
@nmatuschek Please use `!$this->user->isAnonymous()`

Reply:
done.